tests/components/hassio/test_addon_manager.py

```diff
@@ -511,9 +510,8 @@ async def test_update_addon(
     await addon_manager.async_update_addon()
 
     assert addon_info.call_count == 2
-    assert create_backup.call_count == 1
-    assert create_backup.call_args == call(
-        hass, {"name": "addon_test_addon_1.0.0", "addons": ["test_addon"]}, partial=True
+    create_partial_backup.assert_called_once_with(
```

Comment:
I don't recommend using the assert methods on mocks unless the mock is specced. Otherwise a simple typo in the method name can hide an incorrect test.